package.json

```diff
@@ -1,66 +1,67 @@
-    "eslint": "^8.0.0",
-    "mocha": "^9.1.2",
-    "nyc": "^15.1.0",
-    "sinon": "^12.0.1"
-  },
-  "license": "MIT",
-  "engines": {
-    "node": ">=12.0.0"
-  },
-  "scripts": {
-    "pretest": "./node_modules/.bin/eslint lib test index.js",
-    "test": "NODE_ENV=test ./node_modules/.bin/mocha 'test/**/*_test.js'",
-    "test-debug": "NODE_ENV=test ./node_modules/.bin/mocha  --inspect --debug-brk 'test/**/*_test.js'",
-    "test:watch": "NODE_ENV=test ./node_modules/.bin/mocha --watch 'test/**/*_test.js'",
-    "test:coverage": "NODE_ENV=test nyc --reporter=html --reporter=text ./node_modules/.bin/mocha 'test/**/*_test.js'",
-    "lint": "npx eslint .",
-    "lint:fix": "npx eslint . --fix"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/node-oauth/node-oauth2-server.git"
-  },
-  "bugs": {
-    "url": "https://github.com/node-oauth/node-oauth2-server/issues"
-  },
-  "homepage": "https://github.com/node-oauth/node-oauth2-server#readme",
-  "directories": {
-    "doc": "docs",
-    "lib": "lib",
-    "test": "test"
+	"name": "@node-oauth/oauth2-server",
+	"description": "Complete, framework-agnostic, compliant and well tested module for implementing an OAuth2 Server in node.js",
+	"version": "4.1.1",
+	"keywords": [
+	  "oauth",
+	  "oauth2"
+	],
+	"contributors": [
+	  "Thom Seddon <thom@seddonmedia.co.uk>",
+	  "Lars F. Karlström <lars@lfk.io>",
+	  "Rui Marinho <ruipmarinho@gmail.com>",
+	  "Tiago Ribeiro <tiago.ribeiro@gmail.com>",
+	  "Michael Salinger <mjsalinger@gmail.com>",
+	  "Nuno Sousa",
+	  "Max Truxa",
+	  "Daniel Reguero"
+	],
+	"main": "index.js",
+	"types": "index.d.ts",
+	"files": [
+	  "index.js",
+	  "index.d.ts",
+	  "lib"
+	],
+	"dependencies": {
+	  "@node-oauth/formats": "^1.0.0",
+	  "basic-auth": "2.0.1",
+	  "bluebird": "3.7.2",
+	  "promisify-any": "2.0.1",
+	  "type-is": "1.6.18"
+	},
+	"devDependencies": {
+	  "chai": "^4.3.4",
+	  "eslint": "^8.0.0",
+	  "mocha": "^9.1.2",
+	  "nyc": "^15.1.0",
+	  "sinon": "^12.0.1"
+	},
+	"license": "MIT",
+	"engines": {
+	  "node": ">=12.0.0"
+	},
+	"scripts": {
+	  "pretest": "./node_modules/.bin/eslint lib test index.js",
+	  "test": "NODE_ENV=test ./node_modules/.bin/mocha 'test/**/*_test.js'",
+	  "test-debug": "NODE_ENV=test ./node_modules/.bin/mocha  --inspect --debug-brk 'test/**/*_test.js'",
+	  "test:watch": "NODE_ENV=test ./node_modules/.bin/mocha --watch 'test/**/*_test.js'",
+	  "test:coverage": "NODE_ENV=test nyc --reporter=html --reporter=text ./node_modules/.bin/mocha 'test/**/*_test.js'",
+	  "lint": "npx eslint .",
+	  "lint:fix": "npx eslint . --fix"
+	},
+	"repository": {
+	  "type": "git",
+	  "url": "git+https://github.com/node-oauth/node-oauth2-server.git"
+	},
+	"bugs": {
+	  "url": "https://github.com/node-oauth/node-oauth2-server/issues"
+	},
+	"homepage": "https://github.com/node-oauth/node-oauth2-server#readme",
+	"directories": {
+	  "doc": "docs",
+	  "lib": "lib",
+	  "test": "test"
+	}
   }
-}
+  
```

Comment:
Is there a reason for this change in package.json?

Reply:
No, I made a change to run the tests and my vscode setting that formats on save is disabled so it doesn't conflict with the project's lint, ended up going to commit accidentally, I will adjust

Reply:
@dsschiramm thank you